docs/07-physics.md

```diff
@@ -0,0 +1,339 @@
+---
+title: Physics
+path: /docs/physics
+---
+
+Excalibur comes built in with two physics systems. The first system is [[CollisionResolutionStrategy.Box|Box physics]], and is a
+simple axis-aligned way of doing basic collision detection for non-rotated rectangular areas, defined by an actor's
+[[BoundingBox|bounding box]].
+
+### Physics hierarchy
+
+Excalibur physics are organized into a hierarchy, each has a specific single role in the collision system.
+
+```
+Actor (game entity)
+  -> Body (transform information)
+     -> Collider (collision related information)
+        -> Shape (geometry for collision)
+```
+
+For example:
+
+```typescript
+const actor = new ex.Actor({
+  pos: new ex.Vector(100, 100),
+  body: new ex.Body({
+    vel: new ex.Vector(20, 0), // velocity
+    acc: new ex.Vector(0, 100), // acceleration
+    collider: new ex.Collider({
+      mass: 100, // mass of 100
+      type: ex.CollisionType.Active, // active collision type
+      shape: ex.Shape.Circle(50) // circle geometry of radius 50
+    })
+  })
+});
+```
+
+### Actor/Body
+
+Actor's have position, velocity, and acceleration in physical space, all of these positional physical attributes are contained inside the [[Body]]. Only 1 actor can be associated with a [[Body]].
```

Comment:
To avoid the weird Actor's / Body's / Collider's spelling you could make it singular instead. (An actor has ...)